test/cli.js

```diff
@@ -303,6 +305,47 @@ describe('CLI', () => {
         });
     });
 
+    it('starts the inspector with --inspect', (done) => {
+
+        const httpServer = new Http.Server(() => {});
+        httpServer.listen(0, () => {
```

Comment:
Yea. The race condition is with other things on the system that could grab the port.